### DISCUSSION TITLE
Third party application (Antidte, LExibar) doesn't work

### DISCUSSION THREAD

**Discussion by Vernz1337** [SafeExamBrowser/seb-win-refactoring#1057] [Q&A]
Hi,

we are trying to use third party applications such as Antidote, Lexibar. 

The application icon is on the taskbar and start itself as configured but then dissapear.

The antidote window opened but when we click new file, the new window appear 1 sec then dissapear so we cannot copy/paste text.

As for Lexibar, we can see that the application has launched but doesn't read text as intended when we type or select text on the browser.

We only added the application with the .exe path. Do I have to do something else so the application works properly ?

Thanks!

**Comment by Vernz1337** [comment]
I have fixed Antidote but it seems that Lexibar isn't compatible with the SEB browser. It works if we use Chrome tho...

**Reply by GregCSSC** [reply]
Hello, I have same problem with Antidote and Druide can't help me. 
Please can you give us solution ?
you can PM me , sorry for my poor english...

**Reply by GregCSSC** [reply]
Hello
Here are the correct settings so that the Lexibar software works 
![Lexibar](https://github.com/user-attachments/assets/6ed19242-53cb-4bfc-8859-c1ee45de27de)

**Reply by Vernz1337** [reply]
> Hello Here are the correct settings so that the Lexibar software works ![Lexibar](https://private-user-images.githubusercontent.com/189530036/411579830-6ed19242-53cb-4bfc-8859-c1ee45de27de.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA0MjIzNjcsIm5iZiI6MTc1MDQyMjA2NywicGF0aCI6Ii8xODk1MzAwMzYvNDExNTc5ODMwLTZlZDE5MjQyLTUzY2ItNGJmYy04ODU5LWMxZWU0NWRlMjdkZS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDYyMFQxMjIxMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMmZlMWZiYjgwNzlhOTA4ZmU5MDQ0OTAyZTZiMDkyM2NlMTYwZTA1MTA3MDU5NWYxODI4Zjg0ZjM3ZTU0NDhhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.CBV5EVwpU04IIgX-REKbivR5PEGm_KjpUossi2tvEhI)

Still doesn't work, are you using the default SEB Browser?

**Comment by dbuechel** [comment]
Please use the live application log (see https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane > Allow access to application log) and verify whether you potentially need to allow another process. You may also want to try using kiosk mode Disable Explorer Shell. Finally, UWP applications are incompatible with SEB, you'd need to look for an alternative in case that would be the issue.

**Comment by asaraca** [comment]
I have the same problem with WordQ ...

**Reply by GregCSSC** [reply]
Hello
Here are the correct settings so that the WordQ software works
![image](https://github.com/user-attachments/assets/a7004d35-2686-42c9-ad14-c359c7e8091e)

**Reply by RiBust** [reply]
Thank you GregCSSC, I'm going to try this see if i can finally make WordQ6 work with safe exam. It seems on my end is the Runtime Integrity Module that makes a violation integrity compromised alert and WordQ closes with a popup ;(.

**Comment by asaraca** [comment]
Hello,

   Sadly it doesn't work for us.  Error message (in French) "Lecture du texte sélectionné impossible. Votre application ne prend peut-être pas cette fonction en charge."

Antonio Saraca | Administrateur de réseau
[cid:f9923571-a6a6-4e85-a7bb-99609f99647c]      [cid:dea54e30-ef99-4a11-bd34-b8ec9198e402]
425, rue Blainville Est
Sainte-Thérèse (Québec)  J7E 1N7
450 434-1130 , poste x259
Cellulaire : +1 5148312524



Avis de confidentialité
Ce courriel est une communication confidentielle et l’information qu’il contient est réservée à l’usage exclusif du ou de la destinataire. Si vous n’êtes pas la ou le destinataire visé, vous n’avez aucun droit d’utiliser cette information, de la copier, de la distribuer ou de la diffuser. Si cette communication vous a été transmise par erreur, veuillez la détruire et nous en aviser immédiatement par courriel.



________________________________
De : GregCSSC ***@***.***>
Envoyé : 16 juin 2025 13:48
À : SafeExamBrowser/seb-win-refactoring ***@***.***>
Cc : Antonio Saraca ***@***.***>; Comment ***@***.***>
Objet : Re: [SafeExamBrowser/seb-win-refactoring] Third party application (Antidte, LExibar) doesn't work (Discussion #1057)

Vous n’obtenez pas souvent d’e-mail à partir de ***@***.*** Pourquoi c’est important<https://aka.ms/LearnAboutSenderIdentification>

Hello
Here are the correct settings so that the WordQ software works
image.png (view on web)<https://github.com/user-attachments/assets/a7004d35-2686-42c9-ad14-c359c7e8091e>

**Reply by GregCSSC** [reply]
Hello, here are some more precise settings that I made in SEB for Word Q.
First of all I make sure to launch Word Q when starting SEB

<img width="778" alt="Capture d’écran 2025-06-17 082630" src="https://github.com/user-attachments/assets/5f218ed0-d97d-43ca-8a64-8cbe442ebdfe" />
then as usual I make sure that each executable is allowed in SEB
<img width="680" alt="Capture d’écran 2025-06-17 082660" src="https://github.com/user-attachments/assets/c9ef4515-4f9f-47ee-886e-6566c2e521d6" />
We use Word Q5 in our schools, but it should work for other versions. I hope it works.

**Reply by RiBust** [reply]
GregCSSC thank you!  It worked for me with WordQ6.

**Comment by asaraca** [comment]
Thank you,

    I have the exact same setup, but when I try to read aloud some text, I get the error message: "Lecture du texte sélectionné impossible. Votre application ne prend peut-être pas cette fonction en charge".   But everything else works in WordQ.

Antonio Saraca | Administrateur de réseau
[cid:1af9a071-6d30-4138-a74d-d85b42ccc680]      [cid:2bbd070d-d5fa-41db-aa47-8e359578dcf6]
425, rue Blainville Est
Sainte-Thérèse (Québec)  J7E 1N7
450 434-1130 , poste x259
Cellulaire : +1 5148312524



Avis de confidentialité
Ce courriel est une communication confidentielle et l’information qu’il contient est réservée à l’usage exclusif du ou de la destinataire. Si vous n’êtes pas la ou le destinataire visé, vous n’avez aucun droit d’utiliser cette information, de la copier, de la distribuer ou de la diffuser. Si cette communication vous a été transmise par erreur, veuillez la détruire et nous en aviser immédiatement par courriel.



________________________________
De : GregCSSC ***@***.***>
Envoyé : 17 juin 2025 08:35
À : SafeExamBrowser/seb-win-refactoring ***@***.***>
Cc : Antonio Saraca ***@***.***>; Comment ***@***.***>
Objet : Re: [SafeExamBrowser/seb-win-refactoring] Third party application (Antidte, LExibar) doesn't work (Discussion #1057)

Vous n’obtenez pas souvent d’e-mail à partir de ***@***.*** Pourquoi c’est important<https://aka.ms/LearnAboutSenderIdentification>

Hello, here are some more precise settings that I made in SEB for Word Q.
First of all I make sure to launch Word Q when starting SEB

Capture.d.ecran.2025-06-17.082630.png (view on web)<https://github.com/user-attachments/assets/5f218ed0-d97d-43ca-8a64-8cbe442ebdfe> then as usual I make sure that each executable is allowed in SEB Capture.d.ecran.2025-06-17.082660.png (view on web)<https://github.com/user-attachments/assets/c9ef4515-4f9f-47ee-886e-6566c2e521d6> We use Word Q5 in our schools, but it should work for other versions. I hope it works.

**Reply by GregCSSC** [reply]
Here setting for Wordq6, BUT you need to start application before luch SEB 

<img width="731" height="310" alt="image" src="https://github.com/user-attachments/assets/05dd3e9c-72f5-485d-9512-cc28ba5b5ce6" />

I have new problem with Lexibar 3.10 with this action : Blocked 'Ctrl + Injected + C' 
2025-12-15 10:17:43.272 [06] - DEBUG: [ApplicationMonitor] Window has changed from 'Document1 - Word' (196738) to 'Document Microsoft Word' (197144).
2025-12-15 10:18:05.503 [12] - INFO: [KeyboardInterceptor] Blocked 'Ctrl + Injected + RightCtrl' (RightCtrl = 163) when pressed.
2025-12-15 10:18:05.509 [12] - INFO: [KeyboardInterceptor] Blocked 'Ctrl + Injected + C' (C = 67) when pressed.
2025-12-15 10:18:05.514 [12] - INFO: [KeyboardInterceptor] Blocked 'Ctrl + Injected + C' (C = 67) when released.
2025-12-15 10:18:05.524 [12] - INFO: [KeyboardInterceptor] Blocked 'Injected + RightCtrl' (RightCtrl = 163) when released.
I don't know where I can autorise the sequence ?
<img width="264" height="278" alt="image" src="https://github.com/user-attachments/assets/2f590af9-4df1-47e9-b13f-2c09be9856f4" />
